### PR TITLE
**Unbreaking the breaking change** -- rename ariaLabel to aria-label

### PR DESCRIPTION
**Pull request by kevinbarabash**
We'd like to use aria-label instead of ariaLabel so that this prop (and all aria-* in the future) can be consistent between HTML tags and custom components.

Unfortunately, this changes is causing numerous flow errors even though a similar example in the [flow playground](https://flow.org/try/#0PTAEAEDMBsHsHcBQBLAtgB1gJwC6gEoCmAhgMZ6RayqgBEWJ5tA3IojgJ7qGgDKAFsQYATAApV0AZ1ABeUAG8APolCrQwAFQaVa0BtCjikyYWGgcWYgDtJ6IYSs5oHc7HP8evAGoBxAHSgAJKQoFaweOhUAG7IwqYANKDwPEaSAK6oPDgefL6gyJI6avroaQzOoHGk2MQ4yFEpVmbIeAX5VjGSyABG0FlukqQMDgzEcViSfkWqGsDTtELIxAC00MTdhNC0AFygkhbIVgDm8YiKAL7Mutc312yc3AYS0nJK034fAvZiz6e6mtprvoACo5apwLCgWAhbI8ZDVKwBADqyGg0EqhEgxDS0DwODcAANSGUGI4AMKwCEExLwfjw-igTLWaTZWrTPT5PDwVHonDEADWPFgVncPDJvF4oCJlOwBNAUWI0DSPEo1FFoDspLwm0ImUcUyBc104OwAH5dvssIcTtMAez9AB5KxCkKwMqhYiZZoI6Sqmjw4XLVKEHCTABWhUN0wDVl21g4fzUdqBoCdLqlklQiugcoAFABGABs6AAHgBKRIEr3IDJ5gBMABZSxWpWssEdCHmGwAOZuJ4pQyEEkttjt5gCcxfLBt0s2mXQAXoQ41YE7atPa+Jw+izBHhSNZQBsNVRSIRjKYjy4CWNhLxt53QIAUAiP9j2D6h3TDhHIkn7M1AABBdB+CoYQWh4fYOD6T9v1-RJsFAIRLA4XdCAYaEZ2KI01CgvpzSQ1dTguVg2GTfRAJFQIEUqAp0DWVCkL2LM0XaSBsCzOphUVQcMWqSw6gafIsw7JDpEPbx-BUQF9AJOSI2k4TMFwIIEUSeQY0kc5QD9OhwH5QQrDIMZdQ4YB4GFcZll6WBSH5SRlhjFhpMUgAeajhUU3QYxkDSfT8EwhFIfhzi8tQTSwXyKQhPxaQg0KOV0RdCBkWhqwyWgwtUPCUvkeQs3bQ5dgbc4Ev0YAAD5FLkgkXI5JFLPQ0AACE4Ds6RqkyaRuWyJjIGQEtLxMPBoXyH0kIVVF1hg7oXDQZS6mOKVNOpRTujSPAODdUADxFRVJDcSJYBiOJQC290EBFAkPKsQCLxwAkDRkqV5MjfR5uwPAbp0qgaFofTDOMuJUDMiymnQ6y2vsxyEWcjkPpUh4eHkG67uG7TdP+gzrCB0zzMarBIds6GnNYPRFJAUAAFEjBwJrCCOI5dlpWo9kER5PSCU1FJ9fc0n2agbt2VH7tkBQsuY7NdloABZQs-HHOtuzrAAGUAVYAGXzABmPxuwAVkLBs1YbLXdYNo21fzOsNflxXlatwsNatm2TYALVoP59Euaq5Lq2SReGuVJH4N1oDMY9DnVdi0QQa0dI42ptl9gkFI5eRM2zAjLWtRJ0tQbODmORJR0IQurWL0ARyEDty+tMqXtq8mOVBc84XGkOw7MfrWkcNxiHlZBCHgBaoRCItEkbRIe0SawzEnDUBs2P9FIYWwf0EzYOCeuZCBLUe4ixHF9zWYxVOFUA97ppppCIMgcD8UQykICkMGFBwcFc8RYCkCrxaSvkdRSAYiPrib+UgxbyHZOFGUWBpbEiwFqaK2BPbQOysgJc0tM5olQboH20xSTjFzGWf+txeYKAWFaFYawNhbDjFQjW6xNiJAiokGMiQPh+HCB4LA4CtJi2yAUPwR0pBk1uGg0ADAcBlBFLmCRuhXKSCiEceRtxuTCGyL5EqqibgeGQEcfgOAtEJVuKYxY1CmHQF8uYxhtCTGmJuDEYezVYAll8gSNWasAAk8h9baR8X4gk9iHFVQcWoVywAlFHFCbcMsYjVChVCkAA) works fine: .

### CODE REVIEW

**Comment by kevinbarabash** [comment]
Here's an example of some of the other failures I'm seeing when running `yarn flow`:
```
...
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ packages/wonder-blocks-icon-button/components/icon-button.test.js:76:22

Cannot create IconButton element because:
 • property aria-Label is missing in props [1] but exists in SharedProps [2].
 • property aria-label is missing in SharedProps [2] but exists in props [1].

     packages/wonder-blocks-icon-button/components/icon-button.test.js
      73│         const wrapper = mount(
      74│             <MemoryRouter>
      75│                 <div>
 [1]  76│                     <IconButton
      77│                         icon={icons.search}
      78│                         aria-label="search"
      79│                         testId="icon-button"
      80│                         href="/unknown"
      81│                     />
      82│                     <Switch>
      83│                         <Route path="/foo">
      84│                             <div id="foo">Hello, world!</div>

     packages/wonder-blocks-icon-button/components/icon-button.js
 [2] 128│ export default class IconButton extends React.Component<SharedProps> {
...
```

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 51d60f9276603478d5383fa77aeb6d5b8ef4f95d

https://deploy-preview-295--wonder-blocks.netlify.com

File: .flowconfig [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [version]
-0.75.0
+0.79.1
```
Comment (kevinbarabash) [inline .flowconfig]: I thought maybe updating `flow` would alleviate the issues that I was seeing.

File: packages/wonder-blocks-icon/components/icon.js [hunk 2]
```diff
@@ -77,7 +77,14 @@ export default class Icon extends React.PureComponent<Props> {
     };
 
     render() {
-        const {"aria-label": ariaLabel, color, icon, size, style} = this.props;
+        // There is a weird thing where Flow will only recognize a string-quoted
+        // prop name if it's in single quotes, but our tooling normalizes it to
+        // double-quotes on commit. So the aria-label prop isn't included in
```
Comment (kevinbarabash) [inline packages/wonder-blocks-icon/components/icon.js]: @wishdasher thanks for figuring that out.  That's such a bizarre bug.  I assumed flow was converting to an AST before doing its checks.

**Pull request merged**